src/Sylius/Behat/Behaviour/ChoosesName.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Behaviour;
```

Comment:
as well as ``Sylius\Component\Core\Model`` should be renamed to ``Sylius\Component\Core\Models``, as there are many model**s** there, that have many different functionalities? ;)